frontend/src/helpers/utils.js

```diff
@@ -82,14 +83,18 @@ export function reformatChemicalReactionHTML({ reaction, model, noLink = false,
   const addComp = comp || reaction.compartment_str.includes('=>');
   const type = 'metabolite';
   const stoichiometry = x => (Math.abs(x.stoichiometry) !== 1 ? `${x.stoichiometry} ` : '');
+  // superscript + and - at the end of names
```

Comment:
Great that you added superscript for the charge of ions. But I think you may reuse the [chemicalFormula](https://github.com/MetabolicAtlas/MetabolicAtlas/blob/develop/frontend/src/helpers/chemical-formatters.js#L1). With this function, chemicals such as `O2` will also be formatted as O<sub>2</sub>

Reply:
This is an awesome fix 🌟  It resolves my previous comment that it can be weird with `H+ [c] +`.

Reply:
Thanks for the tip, @nanjiangshu ! :+1:  Fixed in f3d70fdf0ab015d2d602b20ff0756a992deb6f51.